tests/unit/functions/updatenotification_spec.js

```diff
@@ -108,13 +108,6 @@ describe("Updatenotification", () => {
 			const { error } = require("logger");
 			expect(error).toHaveBeenCalledWith(`Failed to retrieve repo info for ${moduleName}: Failed to retrieve status`);
 		});
-
-		it("excludes repo if refs don't match regex", async () => {
```

Comment:
why was that removed?

Reply:
see description

the case "no regex match" is now handled different, before an empty object was returned, now another call is done and it is always an object returned. So this test makes no sense anymore.